lib/startUpServer.js

```diff
@@ -39,6 +39,8 @@ function startUpServer(preSessionInject, postSessionInject, appList, appStartLis
     let amorphicOptions = AmorphicContext.amorphicOptions;
     let mainApp = amorphicOptions.mainApp;
     let appContext = AmorphicContext.appContext;
+    let appConfig = AmorphicContext.applicationConfig[mainApp] || {};
```

Comment:
do we ever want to fall back to empty object here?

Reply:
good point I don't think so I was following the pattern elsewhere in the codebase.